mcs/mcs/statement.cs

```diff
@@ -2539,7 +2539,7 @@ public LocalVariable (LocalVariable li, string name, Location loc)
 
 		public bool IsFixed {
 			get {
-				return (flags & Flags.FixedVariable) != 0;
+				return (flags & Flags.FixedVariable) == Flags.FixedVariable;
 			}
 			set {
 				flags = value ? flags | Flags.FixedVariable : flags & ~Flags.FixedVariable;
```

Comment:
`flags & ~Flags.FixedVariable` is wrong now. Assume `flags` is `UsingVariable | FixedVariable== (1 << 7 | 1 << 20) | (1 << 6 | 1 << 20)`, then this expression would evaluate to `1 << 7`.

Reply:
flags can never be `UsingVariable |  FixedVariable` they should not be flags at the first place but I didn't want to mix that into this change